spec/support/token_store/artemis_token.json

```diff
@@ -1 +1 @@
-{"access_token":"eyJhbGciOiJIUzUxMiJ9.eyJpc3MiOiJNeSBBcHAiLCJpYXQiOjE1ODE0OTQ5MDIsImp0aSI6ImIyMjQ3Yzk5LTU5NmUtNDBhMi1iNWZmLTI2ODhlY2Q2ZjVkYyIsInVzZXIiOnsiaWQiOjE2NzMsImVtYWlsIjoiZGFuQGRhbmRlaGF2aWxsYW5kLmNvbSJ9fQ.DIFjEl_fS11BUTq9EnxYTwFGXL2to79dDsmkDz4g5ZFSgVwAlqD_TcILwYz4sbeytiQWZL4a30MzLtaXvl8Gdw","expires_at":1581502102,"refresh_token":"XiozcSUW3PjUl6cpS2PQAbx3EoYHAWsVWGHxtQXt1k0"}
\ No newline at end of file
+{"access_token":"eyJhbGciOiJIUzUxMiJ9.eyJpc3MiOiJNeSBBcHAiLCJpYXQiOjE1ODI4OTk4NzAsImp0aSI6IjdiOWY3NDJlLTU5MjUtNGUxNS1iYmJlLTc3M2QyYjUzZWFhMyIsInVzZXIiOnsiaWQiOjExLCJlbWFpbCI6Im93bmVyQGNhbm5hZGVtby5jb20ifX0.vsV7QkjZUC2-EWv4it0Z_pbliCv3eqFlAL8tpvy2X_58CaJzMoEu_flJ3Zm1MJaNParvSBfcVIn9bTV4_206lA","expires_at":1582907070,"refresh_token":"kbr2kByzfo9j7-D67JyTt4XWrg886pgYLcQpKbJAHrk"}
```

Comment:
I believe we should implement a token generator. Let's leave it for later.

Reply:
I've already added a rake task here: https://github.com/artemis-ag/apps-cannapis/blob/77a43d579f97eb8965ac9f738ff7d4dac0c96e44/lib/tasks/artemis_oauth.rake

Reply:
Awesome!